build-system/babel-config/test-config.js

```diff
@@ -16,18 +16,8 @@
 'use strict';
 
 const argv = require('minimist')(process.argv.slice(2));
-const {getDevDependencies} = require('./dev-dependencies');
```

Comment:
IIRC, this was the only config that actually needed to use what's in `dev-dependencies.js` (even though I added them to other places). It may be possible to remove the `ignore: getDevDependencies()` from all the other babel config files too.